conan/tools/cmake/toolchain/blocks.py

```diff
@@ -955,14 +958,18 @@ def context(self):
         result = self._get_winsdk_version(system_version, generator_platform)
         system_version, winsdk_version, gen_platform_sdk_version = result
 
+        generator_instance = self._conanfile.conf.get("tools.cmake.cmaketoolchain:generator_instance")
+        generator_instance = generator_instance.strip('\'').strip('"') if generator_instance is not None else None
```

Comment:
I think the quoting management would be better addressed more homogeneusly with other confs/options, and not in isolation. If we allow quotes in profiles for this conf, but not for others, it can be confusing and error prone (other confs adding quotes will fail).

I suggest:
- Remove this management of extra quotes in this PR
- Use strings without quotes in profile files in the tests (similar to all other previous confs)

Then we can consider about possible alternatives to make this integrated with ``conf.get()``, so other confs can use it too.

Reply:
Okey, strips removed, we will consider later if we want to support quotations in conan profiles